recipe/meta.yaml

```diff
@@ -13,17 +13,19 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
+    - commonmark >=0.5.6
     - sphinx
```

Comment:
So we have been moving to keep only what is strictly required in `build`. In this case, probably just `python` and `setuptools`.